### PR TITLE
DR-3023: Render static breadcrumbs

### PR DESCRIPTION
**Pull request by 7emansell**
## Ticket:

- JIRA ticket [DR-3023](https://newyorkpubliclibrary.atlassian.net/browse/DR-3023)

## This PR does the following:

- Renders static breadcrumbs

### Checklist:

- [x] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.
- [x] I have updated the CHANGELOG.md.

[DR-3023]: https://newyorkpubliclibrary.atlassian.net/browse/DR-3023?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by 7emansell** [comment]
Breadcrumbs back arrow on mobile is the wrong color, this is getting corrected [here](https://newyorkpubliclibrary.atlassian.net/browse/DSD-1810). Otherwise, good to go and mostly achieved already in the `pageLayout` component PR

**Review by avertrees — APPROVED** [review]

**Comment by avertrees** [comment]
plz update changelog :)

File: src/components/search/search.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```
Comment (EdwinGuzman) [inline src/components/search/search.tsx:1]: wait why?

File: package.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
     "winston": "3.9.0"
   },
   "devDependencies": {
-    "@nypl/design-system-react-components": "3.1.2",
+    "@nypl/design-system-react-components": "3.1.7",
```
Comment (EdwinGuzman) [inline package.json:53]: Can you include this in the changelog as well. The Pr is already merged but in the next PR or in the release branch.

**Pull request merged**